Model/Drink.cs

```diff
@@ -10,25 +10,42 @@ namespace Model
 {
     public enum DrinkType
     {
-        Beer,
-        Wine,
-        Liquor
+        BeerOrPilsener,
+        CraftBeer,
+        IPA,
+        Cider,
+        WhiteWine,
+        RedWine,
+        RoseWine,
+        SparklingWine,
+        Whisky,
+        Rum,
+        SweetLiqour,
+        StrongLiqour,
+        Cocktail,
+        VodkaMix,
+        RumMix,
+        Mix
     }
```

Comment:
Enum naar een los .cs bestand

Reply:
We hebben met de groep besloten dat we deze gewoon in deze klasse houden